.azure/azure-pipeline.yaml

```
@@ -0,0 +1,153 @@
+trigger:
+  branches:
+    include:
+      - main
+      - master
+      - hotfix*
+
+pr:
+  branches:
+    include:
+      - main
+      - master
+      - hotfix*
+
+pool:
+  name: "ch-mg-ue2-agent-pool-mdp001"
+  demands:
+    - agent.os -equals Linux
+    - ImageOverride -equals ubuntu-22.04-covera-build
+
+variables:
+  - group: covera-build
+  - name: CODE_COVERAGE_TARGET_PERCENTAGE
+    value: "80"
+  - name: JFROG_CRITICAL_VULNERABILITY_THRESHOLD_COUNT
+    value: "0"
+  - name: DEVOPS_PROJECT_NAME
```

Please rename this variable to PROJECT_NAME, DEVOPS_PROJECT_NAME is AppDevops_Hub
```suggestion
  - name: PROJECT_NAME
```

---

Done